api/v1alpha1/imagerepository_types.go

```diff
@@ -33,6 +33,11 @@ type ImageRepositorySpec struct {
 	// scans of the image repository.
 	// +optional
 	ScanInterval *metav1.Duration `json:"scanInterval,omitempty"`
+
+	// This flag tells the controller to suspend subsequent kustomize executions,
```

Comment:
Sorry, I should have put my comment about the copypasta here, since the schema is just generated from this type.